package.json

```diff
@@ -1,8 +1,8 @@
 {
-  "name": "translationstudio",
-  "productName": "translationStudio",
-  "version": "0.1.0",
-  "build": "1 alpha",
+  "name": "BTT-Writer",
+  "productName": "BTT-Writer",
+  "version": "1.0.0",
```

Comment:
Too bad we have to start at v 1.0.0  when really this is version 12+  People always get nervous around v1 of any release.